src/components/uploader/js/FileQueueView.js

```diff
@@ -308,9 +309,13 @@ var fluid_2_0 = fluid_2_0 || {};
      */
 
     fluid.defaults("fluid.uploader.fileQueueView", {
-        gradeNames: ["fluid.viewComponent", "autoInit"],
+        gradeNames: ["fluid.viewComponent"],
+        mergePolicy: {
+            //"members.queueFiles": "nomerge"
```

Comment:
Is this here because of the commented-out <code>queueFiles</code> member below and the attached TODO? Is there another way to reflect the needed modelification of this component, rather than leaving commented-out code in?